code/modules/client/preferences.dm

```diff
@@ -1976,3 +1982,24 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			return
 		else
 			custom_names[name_id] = sanitized_name
+
+/// Handles adding and removing donator items from clients
+/datum/preferences/proc/handle_donator_items(client/user)
```

Comment:
I think this is almost more hassle than the config flag is worth. Like this is some serious shitcode if you ask me

Reply:
config flag is for downstreams without a patreon can still use the items silly. This isn't shit code, it's being considerate